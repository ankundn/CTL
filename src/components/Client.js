import React from "react";
import {  MDBRow, MDBCol, MDBCard,  MDBCardBody, MDBIcon, MDBBtn } from "mdbreact";
import "../index.css";


const cocacolalogo = require("../assets/cocacola.png");
const nokialogo = require("../assets/nokia.jpeg");
const microsoftlogo =require("../assets/microsoft.jpg")
const ibmlogo = require("../assets/IBM.png");
const unileverlogo = require("../assets/unilever.jpg");
const wellsfargogrouplogo = require("../assets/wellsfargogroup.png");
const standardcharteredbanklogo = require("../assets/standardchateredbank.png");
const parmalatlogo = require('../assets/parmalat.png')





class Client extends React.Component {
  
  render() {
    return (
      
    
<div className ="mainclientdiv">
  <div>
  <h2 className="h1-responsive font-weight-bold my-5 text-center">
            Our Amazing Clients
          </h2>
          <p className="grey-text w-responsive mx-auto mb-5">
          We establish partnerships, provide responsive service,
          understand client goals, apply intelligent solutions and
          treat clients with respect. 
        We ask that our clients view us as an extension of their teams rather than considering us a
          vendor of services.
          </p>
  </div>
  <div>
    <MDBRow className="text-center">
    <div className="cocacolalogo">
    <img 
                className ="cocacola"
                src= {cocacolalogo}
                alt="Coca Cola Kenya"
              />
            
    </div>
    <div className="ibmlogo">
    <img 
               className ="ibm"
                src= {ibmlogo}
                alt="IBM EA"
              />
              
    </div>
    <div className="nokialogo">
    <img 
               className ="nokia"
                src= {nokialogo}
                alt="Nokia EA"
              />
              
    </div>
    <div className="microsoftlogo">
    <img 
               className ="microsoft"
                src= {microsoftlogo}
                alt="Microsoft EA"
              />           
    </div>  
    </MDBRow>
    <MDBRow>

    <div className="unileverlogo">
    <img 
               className ="unilever"
                src= {unileverlogo}
                alt="Unilever EA"
              />           
    </div>  
    <div className="wellsfargogrouplogo">
    <img 
               className ="wellsfargogroup"
                src= {wellsfargogrouplogo}
                alt="Wells Fargo Group"
              />           
    </div>  
    <div className="standardcharteredbanklogo">
    <img 
               className ="standardcharteredbank"
                src= {standardcharteredbanklogo}
                alt="Standard Chartered Bank"
              />           
    </div> 
    <div className="parmalatlogo">
    <img 
               className ="parmalat"
                src= {parmalatlogo}
                alt="Parmalat SA"
              />           
    </div> 

    </MDBRow>
  </div>

    </div>
    );
  }
}

export default Client;





