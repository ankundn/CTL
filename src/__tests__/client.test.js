import React from "react";
import renderer from "react-test-renderer";
import Client from "../components/Client";

test(" Client renders correctly", () => {
  const tree = renderer.create(<Client />).toJSON();
  expect(tree).toMatchSnapshot();
});

// it('Monday image displays correctly',() => {
//   let MondayImage = renderer.create(<Monday/>).getInstance();
//   expect(MondayImage.change).toMatchSnapshot
// });