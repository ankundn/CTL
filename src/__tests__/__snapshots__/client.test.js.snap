// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Client renders correctly 1`] = `
<div>
  <div
    className="container container z-depth-1 my-5 py-5"
    data-test="container"
  >
    <div>
      <h6
        className="font-weight-bold text-center grey-text text-uppercase small mb-4"
      >
        Clients
      </h6>
      <div
        className="animated bounceIn clientanimation"
        data-test="animation"
        onAnimationEnd={[Function]}
        onAnimationIteration={[Function]}
        onAnimationStart={[Function]}
        style={
          Object {
            "animationDelay": undefined,
            "animationDuration": "3s",
            "animationIterationCount": 1,
            "animationName": "bounceIn",
            "visibility": "visible",
          }
        }
      >
        <h2
          className="font-weight-bold text-center dark-grey-text pb-2"
        >
           Meet Some of Our Amazing Clients
        </h2>
      </div>
      <hr
        className="w-header my-4"
      />
      <p
        className="text-center w-responsive mx-auto pb-5"
      >
        We establish partnerships, provide responsive service, understand client goals, apply intelligent solutions and treat clients with respect. We ask that our clients view us as an extension of their teams rather than considering us a vendor of services.
      </p>
      <div
        className="animated bounceIn clientanimation"
        data-test="animation"
        onAnimationEnd={[Function]}
        onAnimationIteration={[Function]}
        onAnimationStart={[Function]}
        style={
          Object {
            "animationDelay": undefined,
            "animationDuration": "3s",
            "animationIterationCount": 1,
            "animationName": "bounceIn",
            "visibility": "visible",
          }
        }
      />
    </div>
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-12 col-lg-4 cocacolalogo"
        data-test="col"
      >
        <img
          alt="Coca Cola Kenya"
          className="img-fluid z-depth-1"
          src="cocacola.png"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-1-half"
          src="nokia.jpeg"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-2"
          src="microsoft.jpg"
        />
      </div>
    </div>
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-12 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-3"
          src="IBM.png"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-4"
          src="unilever.jpg"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-5"
          src="standardchateredbank.png"
        />
      </div>
    </div>
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-12 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-3"
          src="parmalat.png"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-4"
          src="caterpillargroup.jpg"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-5"
          src="enterpriserentacar.jpg"
        />
      </div>
    </div>
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-12 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-3"
          src="KDN.jpg"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-4"
          src="NMG.jpg"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-5"
          src="bestnetltd.png"
        />
      </div>
    </div>
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-12 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-3"
          src="tata.png"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-4"
          src="CandRGroup.png"
        />
      </div>
      <div
        className="col-md-6 col-lg-4 mb-4"
        data-test="col"
      >
        <img
          alt=""
          className="img-fluid z-depth-5"
          src="phoenixaviation.jpeg"
        />
      </div>
    </div>
  </div>
</div>
`;
